api/jobs/jobs.py

```diff
@@ -83,10 +125,14 @@ def load(cls, e):
         # Don't modify the map
         d = copy.deepcopy(e)
 
-        if d.get('inputs', None):
-            inputs = d['inputs']
-            for x in inputs.keys():
-                inputs[x] = create_filereference_from_dictionary(inputs[x])
+        if d.get('inputs'):
```

Comment:
I'm not sure why we have it everywhere, as `None`, `''`, `{}` and `[]` are all falsey in Python. I'm going to assume I might have put it a few places before I knew better :).